packages/example-app-base/src/utilities/extractAnchorLink.ts

```diff
@@ -1,10 +1,11 @@
 /**
  * Returns the last hash value in the link
  */
-export function extractAnchorLink(url: string): string | undefined {
-  const split = url.split('#');
+export function extractAnchorLink(url: string): string {
+  const split = (url || '').split('#');
```

Comment:
Should the arg type be `url?: string`?

Reply:
The URL really ought to always be defined (and it turned out the source of the bug was the function's output being undefined, not its input). I reverted adding a null check on the input since that arguably should help make it more explicit if there's a bug.